libs/gamepad/src/GamepadHeaders/Buttons.h

```diff
@@ -1,9 +1,10 @@
 #pragma once
 
 #include "GamepadInput.h"
+#include <array>
 
 namespace gamepad {
-  static GamepadInput Buttons[32] = {
+  static constexpr auto Buttons = std::array<GamepadInput, 32>{
```

Comment:
I disabled the no C style arrays check. Is there any reason to prefer `std::array` here? Doesn't seem to add value.

Also, what about:
```cpp
static constexpr std::array<GamepadInput, 32> Buttons { ... }
```

Reply:
oh. lmao. I just read the description. Work did this to me. I'm not used to seeing PR descriptions. LOL.

But can't it be `constexpr GamepadInput Buttons[32]`?